lib/rules/no-restricted-modules.js

```diff
@@ -65,8 +65,7 @@ module.exports = {
                             patterns: arrayOfStrings
                         },
                         additionalProperties: false
-                    },
-                    additionalItems: false
+                    }
```

Comment:
This should have been a 1-element tuple, but since it's a deprecated rule it's fine to not change anything.

Reply:
there were couple of other places - probably worth making them all 1-tuples if that was the intention?

Reply:
I'll check more in detail. Some seem to be just useless, some look as if they should be 1-tuples.

If we decide to merge some fixes for a minor version, those fixes would be probably only removing useless `additionalItems`. Per our semver policy, a schema bug fix in a minor version shouldn't break end users, even if their config has de facto invalid settings.

Reply:
I checked all removed `additionalItems: false`, and this is the only one that was supposed to limit the number of elements in an array. Others were just useless.

Given that this rule is deprecated, it's fine to remove the ignored `additionalItems: false` and not fix the schema.